extensions/adobe/experience/campaign-experienceevent-v2.schema.json

```diff
@@ -0,0 +1,172 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/campaign-experienceevent-v2",
```

Comment:
The change from our side would be to just update the mixin ref with the new one (containing -v2), is that correct?

Reply:
Correct but there are additional changes in the v2 versions.  I modified the mutuation event schema so that you lose the hierarchical mess when trying to reference the message field (see pic)
![image](https://user-images.githubusercontent.com/6667779/59929278-1e440680-9406-11e9-9aad-6f78bb7b6071.png)

I also dropped the web object within mutation event.

Reply:
Yes got it thanks